### DISCUSSION TITLE
Compile time error

### DISCUSSION THREAD

**Discussion by KormiMeiko** [misskey-dev/misskey#14012] [General]
## 💡 Summary
An error occurred while compiling

## 🥰 Expected Behavior

The task buildx succeeds and is published to dockerhub

## 🤬 Actual Behavior
github action shows:

------
Dockerfile:13
--------------------
  12 |     
  13 | >>> RUN apk add --no-cache $BUILD_DEPS && \
  14 | >>>     git submodule update --init && \
  15 | >>>     yarn install && \
  16 | >>>     yarn build && \
  17 | >>>     rm -rf .git
  18 |     
--------------------
error: failed to solve: process "/bin/sh -c apk add --no-cache $BUILD_DEPS &&     git submodule update --init &&     yarn install &&     yarn build &&     rm -rf .git" did not complete successfully: exit code: 1
Error: buildx failed with: error: failed to solve: process "/bin/sh -c apk add --no-cache $BUILD_DEPS &&     git submodule update --init &&     yarn install &&     yarn build &&     rm -rf .git" did not complete successfully: exit code: 1

## 📝 Steps to Reproduce

1.download misskey from https://github.com/misskey-dev/misskey/
2.edit language and image
3.upload

## 📌 Environment

Misskey version:12.108.1
Your OS:Windows10
Your browser:google chrome

**Comment by ThatOneCalculator** [comment]
I don't know too much about Windows, but isn't `buildx` the .net/csharp compiler? This is a NodeJS program.

**Comment by KormiMeiko** [comment]
> I don't know too much about Windows, but isn't `buildx` the .net/csharp compiler? This is a NodeJS program.

i just use the github action's docker image workflow

**Comment by ThatOneCalculator** [comment]
> i just use the github action's docker image workflow

You should follow the instructions in the documentation instead: https://misskey-hub.net/en/docs/install.html
